src/docs/patterns/appbar.mdx

```diff
@@ -1,43 +1,86 @@
+
+<React.Fragment>
+    <ImageGrid
+        regularWidth
+        images={[DropdownVariations]}
+        caption={
+            ('1. On mobile, the dropdown toolbar opens the options panel in a bottom sheet.',
+            '2. On desktop, the dropdown opens as a menu below the toolbar.')
+        }
+    />
+    <DemoCard
+        repository={'Dropdown Toolbar'}
+        angular={
+            'https://pxblue-components.github.io/angular/?path=/story/components-dropdown-toolbar--with-full-config'
+        }
+        react={'https://pxblue-components.github.io/react/?path=/story/components-dropdown-toolbar--with-full-config'}
+        ionic
+        reactNative
+    />
+    <Divider />
+</React.Fragment>
+
+# Collapsible App Bar
 
 <ImageGrid
     images={[Image1, Image2]}
     caption={'A collapsible app bar shrinks and gets pinned to the top when the page is scrolled.'}
 />
 <DemoCard repository={'collapsible-appbar'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Fapp-bar%2Fcollapsible-app-bar%2Fcollapsible-app-bar.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/collapsible-app-bar/index.tsx&initialpath=collapsible'} ionic reactNative />
 
-In some situations, you may want to present more information in an app bar, such as a large graphic, buttons, or multiple lines of text. In these cases, you can use a taller app bar that will shrink as the user scrolls down the page and be replaced by a basic app bar. This allows you the flexibility to present an eye-catching banner without permanently reducing the amount of usable space on the page. This is a pattern most commonly used for mobile applications, but could also be used at a larger scale.
+In some situations, you may want to present more than just a title and subtitle in an app bar, such as a large graphic, buttons, or multiple lines of text. In these cases, you can use a taller app bar that will shrink as the user scrolls down the page until it becomes a basic app bar. This allows you the flexibility to present an eye-catching banner without permanently reducing the amount of usable space on the page. This is a pattern most commonly used for mobile applications, but could also be used for web.
 
-## Search Bar
+The collapsible appbar should transition smoothly between the expanded and collapsed states — this means you will need to account for how the app bar will look throughout the transition. Because elements will disappear as the app bar shrinks, you'll need to use dynamic text scaling, opacity transitions, and / or other techniques to do this smoothly, depending on your content.
```

Comment:
We shall probably be more opinionated on this. Ideally, it should look like the Header component in React Native Component Library, but at the same time, I understand that this pattern can be a bit tricky in implementing the animation.

Reply:
It depends on the content too. Are we telling people what to put inside of it (i.e, a collapsible app bar has three lines of text and a background image) or are we leaving that up to them and saying the pattern is just "big toolbar becomes small and make the content transition cleanly" and they could put anything they want in it?...I was assuming the latter.

Reply:
I see what you mean here. I personally think we shall encourage the use of animations like RN Header bar by including a pic of it, then we can mention that there are several other different ways you can animate it